runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsDestinationOverlay.java

```diff
@@ -36,37 +37,43 @@
 import net.runelite.client.ui.overlay.OverlayPriority;
 import net.runelite.client.ui.overlay.OverlayUtil;
 
-public class TileIndicatorsOverlay extends Overlay
+import javax.inject.Inject;
+
+public class TileIndicatorsDestinationOverlay extends Overlay
 {
 	private final Client client;
 	private final TileIndicatorsConfig config;
 
-	TileIndicatorsOverlay(Client client, TileIndicatorsConfig config)
+	@Inject
+	private TileIndicatorsDestinationOverlay(Client client, TileIndicatorsConfig config)
 	{
-		this.client = client;
-		this.config = config;
 		setPosition(OverlayPosition.DYNAMIC);
 		setLayer(OverlayLayer.ABOVE_SCENE);
 		setPriority(OverlayPriority.LOW);
+
+		this.client = client;
+		this.config = config;
```

Comment:
Why this change?

Reply:
For consistency in relation to the PositionOverlay.